tailwind.config.js

```diff
@@ -3,7 +3,10 @@ module.exports = {
   content: ["./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
     // Omit Tailwind colors. Use DaisyUI colors instead
-    colors: {},
+    colors: {
+      "neutral-light":"#E8F0FE",
```

Comment:
We should **not** extend colors here. These should be added in the daisyUI themes section (for both `light` and `dark` themes)